integration/docker/entrypoint.sh

```diff
@@ -47,20 +47,26 @@ alluxio_env_vars=(
 declare -A alluxio_env_violators=(
```

Comment:
@bf8086 why not move violators logic to java land as well?

Reply:
Done.